chefignore

```diff
@@ -92,3 +92,10 @@ Vagrantfile
 # Travis #
 ##########
 .travis.yml
+
+# chef-provisioner
+.chef/*.pem
+.chef/local-mode-cache/
+/clients/
+/nodes/
+/vms/
```

Comment:
We can probably move both of those files into a ``chef-provisioning`` folder. Are there any other files you'd like to move there?